src/node_crypto.cc

```diff
@@ -5422,7 +5413,7 @@ void PBKDF2(const FunctionCallbackInfo<Value>& args) {
   }
 
   raw_keylen = args[3]->NumberValue();
-  if (raw_keylen < 0.0 || isnan(raw_keylen) || isinf(raw_keylen) ||
+  if (raw_keylen < 0.0 || std::isnan(raw_keylen) || std::isinf(raw_keylen) ||
```

Comment:
Since `algorithm` header is added above and `cmath` is included in algorithm, the standard `isnan` and `isinf` macros are undefined and new functions are introduced under `std` namespace. Therefore, need to use `std` namespace for these 2 functions.
You can see more detailed information in `cmath` header.